content/posts/fedora-eln.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "Working on the next major RHEL release, in your upstream repo"
+date: 2020-10-04T12:07:03+02:00
+draft: false
+---
+
+[Fedora EL Niño](https://docs.fedoraproject.org/en-US/eln/) (ELN) is such an
+awesome idea. It enables building rawhide packages in two distinct buildroots:
+
+1. the standard Fedora Rawhide buildroot
+2. and the second one, which mimics Red Hat Enterprise Linux
+
+This way you can make sure that your new upstream release builds fine in the
+next RHEL.
+
+It's a little unfortunate we are getting this feedback too late: the upstream
+release already happened and the code was imported in Fedora dist-git, so
+fixing an issue will require repeating the whole process. Wouldn't it be better
+to know if the upstream change builds fine in ELN **while** working on the
+code?
+
+Oh, wait!
+
+### You can do this easily with Packit
+
+If your GitHub project is not using Packit yet, [here's a
+guide](https://packit.dev/packit-as-a-service/) how to start.
+
+Once it's set up, you need to make sure that your pull requests are also being
+built in the `fedora-eln` target:
+
+```
+jobs:
+  - job: copr_build
+    trigger: pull_request
+    metadata:
+      targets:
+        - fedora-development
+        - fedora-eln
+```
+
+With this config, changes from every pull request will be built in all
+development versions of Fedora (at the time of writing this, it's Rawhide and
+Fedora 33) and in Fedora ELN.
+
+Easy, right?
+
+You can set up the same thing for pushes to a branch, for example build the
+master branch in ELN and development versions of Fedora:
```

Comment:
```suggestion
You can set up the same to happen when pushing to a branch, for example build the
`master` branch in ELN and development versions of Fedora:
```

Reply:
I disagree on this one - I prefer my wording since it's shorter and I feel like the word "happen" is implied

Reply:
since we can't agree, I'll rewrite it completely

Reply:
The part that does not sound right to me is "pushes to a branch".